test/unit/cute/intel_xe/copy_block.cpp

```diff
@@ -0,0 +1,1817 @@
+        Layout<Shape<_1, _16>, Stride<_0, _1>>{},
+        Layout<Shape<_1, _1>, Stride<_1, _0>>{});
+    auto tiled_store = make_tiled_copy(
+        Copy_Atom<Copy_Traits<XE_2D_U32x1x16_ST_N>, dtype>{}.with(device_output, N, M,
+                                                           N),
+        Layout<Shape<_1, _16>, Stride<_0, _1>>{},
+        Layout<Shape<_1, _1>, Stride<_1, _0>>{});
+    static constexpr auto subgroup_size = 16;
+    auto blockDim = syclcompat::dim3(size(tiled_load));
+//
+// Launch the kernel
+//
+    syclcompat::experimental::launch<
+        copy_kernel_vectorized<decltype(S), decltype(D), decltype(tiled_load),
+                               decltype(tiled_store), XE_2D_U32x1x16_LD_N>,
+        subgroup_size>(1, blockDim, S, D, tiled_load, tiled_store);
+
+    syclcompat::wait_and_throw();
+    syclcompat::memcpy<dtype>(host_output.data(), device_output, M * N);
+    for (int i = 0; i < M * N; ++i) {
+      // printf("%d  %d\n", int(h_in[i]), int(h_out[i]));
+      EXPECT_EQ(host_output[i], host_src[i]);
+    }
+    print("XE_2D_U32x1x16_LD_N test end \n\n");
+  }
+
+  {
+    print("XE_2D_U32x2x16_LD_N test: \n");
+    constexpr int M = 2;
+    constexpr int N = 16;
+    using dtype = uint32_t;
+    //
+    // Allocate and initialize
+    //
+    std::vector<dtype> host_src(M * N);
+    std::vector<dtype> host_output(M * N);
+
+    dtype *device_src = syclcompat::malloc<dtype>(M * N);
+    dtype *device_output = syclcompat::malloc<dtype>(M * N);
+
+    for (size_t i = 0; i < host_src.size(); ++i) {
+      host_src[i] = static_cast<dtype>(i);
+    }
+
+    syclcompat::memcpy<dtype>(device_src, host_src.data(), M * N);
+    syclcompat::memcpy<dtype>(device_output, host_output.data(), M * N);
+
+    Tensor S =
+        make_tensor(make_gmem_ptr(device_src),
+                    make_layout(Shape<Int<M>, Int<N>>{}, Stride<Int<N>, _1>{}));
+    Tensor D =
+        make_tensor(make_gmem_ptr(device_output),
+                    make_layout(Shape<Int<M>, Int<N>>{}, Stride<Int<N>, _1>{}));
+
+    auto tiled_load = make_tiled_copy(
+        Copy_Atom<Copy_Traits<XE_2D_U32x2x16_LD_N>, dtype>{}.with(device_src, N, M, N),
+        Layout<Shape<_1, _16>, Stride<_0, _1>>{},
+        Layout<Shape<_2, _1>, Stride<_1, _0>>{});
+    auto tiled_store = make_tiled_copy(
+        Copy_Atom<Copy_Traits<XE_2D_U32x2x16_ST_N>, dtype>{}.with(device_output, N, M,
+                                                           N),
+        Layout<Shape<_1, _16>, Stride<_0, _1>>{},
+        Layout<Shape<_2, _1>, Stride<_1, _0>>{});
+    static constexpr auto subgroup_size = 16;
+    auto blockDim = syclcompat::dim3(size(tiled_load));
+//
+// Launch the kernel
+//
+    syclcompat::experimental::launch<
+        copy_kernel_vectorized<decltype(S), decltype(D), decltype(tiled_load),
+                               decltype(tiled_store), XE_2D_U32x2x16_LD_N>,
+        subgroup_size>(1, blockDim, S, D, tiled_load, tiled_store);
```

Comment:
This launch definition is not valid any more, and we must use launch with properties to specify the sub group size. 
Do the tests compile ?

Reply:
Also, the way the launch range is provided is wrong, i.e `1` instead of `dim3(1)`